lib/casbin-ruby-sql-adapter/adapter.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'sequel'
+require 'casbin-ruby'
+
+module CasbinRubySqlAdapter
+  # the interface for Casbin adapters
+  class Adapter < Casbin::Persist::Adapter
+    attr_reader :db, :table_name, :filtered
+
+    def initialize(db_url:, db_options: {}, table_name: :casbin_rule, filtered: false)
+      @table_name = table_name.to_sym
+      @filtered = filtered
+      @db = Sequel.connect(db_url, db_options)
+      migrate
```

Comment:
I think, we should remove migrate action from initialization. Maybe let users to use migration as task (sdk method) from  frameworks they used?